src/main/java/AddCommand.java

```diff
@@ -0,0 +1,88 @@
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+
+/**
+ * Class for the command that adds a task into the tasklist.
+ */
+public class AddCommand extends Command {
+
+    private String input;
+
+    /**
+     * Constructor for the AddCommand object.
+     *
+     * @param ipt the string representation of the command input.
+     */
+    public AddCommand(String ipt) {
+        input = ipt;
+    }
+
+    /**
+     * Indicates whether this is an exit command.
+     *
+     * @return false.
+     */
+    public boolean isExit() { return false; }
```

Comment:
Would be clearer and improve readability to have your return statement on a new line!
Maybe you can try running the CheckStyle tool? 
Would help to identify these issues!